apps/web/src/design-system/switch/Switch.styles.ts

```diff
@@ -0,0 +1,26 @@
+import { createStyles, MantineTheme } from '@mantine/core';
+
+export default createStyles((theme: MantineTheme, _params, getRef) => {
+  const dark = theme.colorScheme === 'dark';
+  const label = getRef('label');
+  return {
+    input: {
+      backgroundColor: dark ? theme.colors.dark[5] : theme.colors.gray[5],
+      borderColor: 'transparent',
+      '&:checked': {
+        backgroundImage: theme.colors.gradient[8],
+        [`& + .${label}`]: {
+          backgroundImage: theme.colors.gradient[8],
+          backgroundClip: 'text',
+          color: 'transparent',
+        },
+      },
+    },
+    label: {
+      ref: label,
+      paddingLeft: '10px',
+      backgroundImage: 'none',
+      color: dark ? theme.colors.dark[6] : theme.colors.gray[6],
+    },
+  };
```

Comment:
In disabled mode there is some border around the circle, I think it shouln't be there
![CleanShot 2022-02-06 at 19 19 28](https://user-images.githubusercontent.com/8872447/152692678-05c9b189-2822-4007-8f0f-ce95908d7a03.png)